src/store/providers/ccxtServerProvider.ts

```
@@ -101,6 +104,172 @@ export class CCXTServerProviderImpl {
+   * Устанавливает WebSocket соединение с сервером
+   */
+  private async connectWebSocket(): Promise<Socket> {
+    if (this.socket && this.socket.connected) {
+      return this.socket;
+    }
+
+    console.log(`🔌 [CCXTServer] Connecting to WebSocket: ${this.baseUrl}`);
+
+    this.socket = io(this.baseUrl, {
+      transports: ['websocket'],
+      timeout: this.timeout,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)